windows_build.rs

```diff
@@ -32,18 +34,22 @@ impl<'a> Provider<'a> {
 }
 
 impl<'a> PHPProvider<'a> for Provider<'a> {
-    fn new(info: &'a PHPInfo) -> Result<Self> {
-        let version = info.version()?;
-        let is_zts = info.thread_safety()?;
-        let arch = info.architecture()?;
-        let devel = DevelPack::new(version, is_zts, arch)?;
+    fn new(build: &'a Build) -> Result<Self> {
+        // don't use `build.version.to_string()` as it includes extra part which is not needed.
+        let version = format!(
+            "{}.{}.{}".build.version.major,
+            build.version.minor, build.version.release
+        );
```

Comment:
`version.to_string()` could return something like `8.2.0RC6`, which is not something we want.